<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="bloodCheckBox">
       <property name="text">
        <string>无限血量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="shellCheckBox">
       <property name="text">
        <string>无限防弹衣</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="moneyCheckBox">
       <property name="text">
        <string>无限金钱</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bulletCheckBox">
       <property name="text">
        <string>无限子弹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="grenadeCheckBox">
       <property name="text">
        <string>无限手雷（需提前买）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="recoilCheckBox">
       <property name="text">
        <string>辅助压枪</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>未检测到游戏运行...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
